src/store.ts

```diff
@@ -34,6 +34,7 @@ import {
 import { IMinifiedConsentJSONObject, SDKConsentState } from './consent';
 import { Kit, MPForwarder } from './forwarders.interfaces';
 import {
+    CookieSyncDate,
```

Comment:
Could you make this plural in the interfaces so it's `CookieSyncDates`?  Seems related and quick enough to not be _totally_ out of scope...but up to you.